mycroft/client/enclosure/mark1/__init__.py

```diff
@@ -135,7 +134,7 @@ def process(self, data):
             time.sleep(0.5)  # Prevents recording the loud button press
             record(os.path.join(tempfile.gettempdir(), 'test.wav'), 3.0)
             mixer.setvolume(prev_vol)
-            play_wav(os.path.join(tempfile.gettempdir(), 'test.wav').communicate()
+            play_wav(os.path.join(tempfile.gettempdir(), 'test.wav').communicate())
```

Comment:
Not sure about this but I think it should be
```python
            play_wav(os.path.join(tempfile.gettempdir(), 'test.wav')).communicate()
```

since play_wav returns a subprocess object we want to communicate with to ensure it's finished.